codespell_lib/data/dictionary.txt

```diff
@@ -10107,6 +10141,14 @@ doign->doing
 doiing->doing
 doiuble->double
 doiubled->doubled
+dokc->dock
+dokced->docked
+dokcer->docker
+dokcerd->dockerd
+dokcing->docking
+dokcre->docker
+dokcred->dockerd
```

Comment:
Should this (and `dokcerd`) have the same `->dockerd, docked, docker,` correction as `dcokerd` ?